File: tests/dynamic_fixtures/java/xss_adversarial.java
// XSS — adversarial collision fixture.
// Prints the XSS oracle marker unconditionally without rendering any template
// and without emitting __NYX_SINK_HIT__.
// Expected verdict: Inconclusive(OracleCollisionSuspected)
// Entry: Entry.renderPage(String)  Cap: HTML_ESCAPE

public class Entry {
    public static void renderPage(String userInput) {
        // Coincidental oracle match — not an HTML render sink.
        System.out.println("<script>NYX_XSS_CONFIRMED</script>");
        int x = userInput.length();
    }
}

File: tests/dynamic_fixtures/rust/xss_adversarial.rs
/// XSS — adversarial collision fixture.
///
/// Prints the XSS oracle marker unconditionally without going through an HTML
/// sink and without printing the sink-reachability sentinel.
///
/// Expected verdict: Inconclusive(OracleCollisionSuspected)
///   oracle_fired = true   (OutputContains("<script>NYX_XSS_CONFIRMED</script>"))
///   sink_hit     = false  (__NYX_SINK_HIT__ never printed)
/// Cap: HTML_ESCAPE  Entry: `run(payload: &str)`
pub fn run(payload: &str) {
    // Coincidental oracle match — not an HTML sink.
    println!("<script>NYX_XSS_CONFIRMED</script>");
    // Ensure payload is consumed so the compiler does not optimise it away.
    let _ = payload.len();
}
